lib/validators/properties-validator.ts

```diff
@@ -88,4 +91,17 @@ export class PropertiesValidator extends Validator {
             }
         }
     }
+
+    private validateChildProperties(
+        property: ComponentProperty,
+        componentPropertyNames: Set<string>,
+        component: ParsedComponent,
+    ) {
+        (property.childProperties || []).forEach((conditionalChildProperties) => {
+            const conditionalUsedProperyNames = new Set(componentPropertyNames);
```

Comment:
Typo: `conditionalUsedPropertyNames`. This const can also be defined above the `forEach` I think?

Reply:
It can't be moved, it needs to be reset for every conditional child properties object. Otherwise you can't use the same property in another conditional child properties object